### ISSUE TITLE
[pni] cross page: ensure landmark elements exist

### ISSUE THREAD

**Issue by Pomax** [mozilla/foundation.mozilla.org#4641]
We should have `<header>`, `<main>` and `<footer>` at the least, but we currently only have that last one.
